systemtest/src/test/java/io/strimzi/systemtest/tracing/TracingST.java

```diff
@@ -111,6 +113,9 @@
 
     @Test
     void testProducerService() {
+        // TODO temporary fix
```

Comment:
Again, lot of these temporary fixes ... it is not clear how temporary they are and if they should still be here.